<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionLabelerWidget</class>
 <widget class="QWidget" name="ActionLabelerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionLabelerWidget</string>
  </property>
  <widget class="QListWidget" name="actionListWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>171</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Label Frames</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addSelectToCurrButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Add select to current frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearCurrLabelButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Clear current label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="fromFrameNumEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="toFrameNumEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addSelectToFramesButton">
      <property name="text">
       <string>Add select to frames</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearLabeOfFramesButton">
      <property name="text">
       <string>Clear labels</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearAllLabelButton">
      <property name="text">
       <string>Clear all labels</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Frame</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="frameIdLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="frameLabelsLabel">
      <property name="text">
       <string>Label:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="frameLabelWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="frameBrowseSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="saveLabelButton">
      <property name="text">
       <string>Save label file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="modelNameListWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clearSelectionButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear selection</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
